src/Bybit/Spot/Spot.jl

```diff
@@ -0,0 +1,20 @@
+module Spot
+
+using CryptoAPIs.Bybit
+
+"""
+    public_client = BybitClient(; base_url = "https://api.bybit.com")
+"""
+const public_client =
+    BybitClient(; base_url = "https://api.bybit.com")
+
+include("API/Candle.jl")
+using .Candle
+
+include("API/SymbolsInfo.jl")
+using .SymbolsInfo
+
+include("API/Deposit.jl")
+using .Deposit
```

Comment:
it is necessary to sort by method name
```suggestion
include("API/Deposit.jl")
using .Deposit

include("API/SymbolsInfo.jl")
using .SymbolsInfo
```